common/FileSystem.cpp

```diff
@@ -284,6 +284,95 @@ bool Path::IsAbsolute(const std::string_view path)
 #endif
 }
 
+bool Path::EscapeCmdLine(std::string* arg)
+{
+#ifdef _WIN32
+	if (arg->empty() || arg->find_first_of(" \t\"") == std::string::npos)
```

Comment:
This logic still looks wrong (see my previous comment), although honestly I would wait for a Windows person to review this at this point.